docs/config/settings.md

```diff
@@ -85,3 +93,10 @@ Roc assumes that the validators used is either a RegExp or a function that will
 For convenience several types of validators exists in `roc` that can be imported from `roc/validators`. For a complete list of them please see [the JSDocs](/docs/JSDocs.md).
 
 See [roc.config.meta.js in roc-web for an example of how the configuration might look](https://github.com/vgno/roc-web/blob/master/src/roc/config/roc.config.meta.js#L62).
+
+### `converters`
+Roc will by default provide an automatic converter that will base its converter on the default value of the settings. In most cases will this be sufficient but in some instances a custom converter might be needed. This custom converter will in most cases get a string as input and it's expected to return the correct type.
```

Comment:
Roc will by default provide an automatic converter that will be based on the default value of the corresponding settings. In most cases will this be sufficient but in some instances a custom converter might be needed. This custom converter will in most cases get a string as input and be expected to return the correct type.